docs/book/cookbook/aot-guide.md

```diff
@@ -0,0 +1,112 @@
+# Using AoT In A ZendFramework App
+
+This guide will show you how you can use zend-di's AoT compiler
+to make your ZendFramework application production ready.
+
+You will learn how to:
+
+* Add a script to run the compilation
+* Use the generated injector with ServiceManager
+* Use the generated factories with ServiceManager
+
+## Create project and add zend-di
+
+For this guide will use an [expressive application](https://docs.zendframework.com/zend-expressive/)
```

Comment:
Is this a "zend-expressive application" or "Zend Framework application"? I think there's a difference.

Reply:
I thought expressive was part of zend framework and an alternative to mvc. Anyway it's a good idea to change the title to make it more clear we're dealing with expressive here. I'll rephrase this paragraph accordingly.